public/scss/_base.scss

```diff
@@ -76,6 +76,62 @@ li:not(:last-of-type) {
   @extend %page-container;
 }
 
+.fip-container {
+  flex-direction: column;
+  display: flex;
+
+  @include sm {
+    justify-content: space-between;
+    flex-direction: row;
+  }
+}
+
+.phase-banner {
+  background: #F4F4F4;
```

Comment:
Would one of our other colours work? If not, it's fine. If it's just in this one place, we're probably okay.